README.md

```diff
@@ -72,11 +73,12 @@ You can optionally copy the contents of `.env.example` to a `.env` file that wil
 - `LAKEHOUSE_ABFSS_PREFIX`: The prefix for the Azure Blob File System Storage (ABFSS) path.  Should match pattern `abfss://<workspace_id>@msit-onelake.dfs.fabric.microsoft.com/<lakehouse_id>`.  Get this value by selecting "Properties" on your Lakehouse Tables location and copying "ABFS path".
 - `DPS_TABLE_NAME`: The name of the table where data point values and timestamps should be stored in Fabric.  The replicator will create the table if it does not exist.
 - `TS_TABLE_NAME`: The name of the table where time series metadata should be stored in Fabric.  The replicator will create the table if it does not exist.
+- `EVENT_TABLE_NAME`: The name of the table where event data should be stored in Fabric. The replicator will create the table if it does not exist.
 
 ## Fabric Extractor Variables
-- `EXTRACTOR_EVENT_PATH`: The ABFSS file path for the events table in a Fabric lakehouse.
-- `EXTRACTOR_FILE_PATH`: The ABFSS file path for the files in a Fabric lakehouse.
-- `EXTRACTOR_RAW_TS_PATH`: The ABFSS file path for the gittimeseries table in a Fabric lakehouse.
+- `EXTRACTOR_EVENT_PATH`: The table path for the events table in a Fabric lakehouse.  It's the relative path after the ABFSS prefix (e.g. "Tables/RawEvents")
+- `EXTRACTOR_FILE_PATH`: The file path for the files in a Fabric lakehouse. It's the relative path after the ABFSS prefix (e.g. "Files/Tanks.png")
+- `EXTRACTOR_RAW_TS_PATH`: The file path for the gittimeseries table in a Fabric lakehouse. It's the relative path after the ABFSS prefix (e.g. "Tables/RawTS")
```

Comment:
What is "gittimeseries table"?

Reply:
It should be a typo. Fixed to "raw timeseries"